vocabularies/Offline.xml

```diff
@@ -49,7 +49,21 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
            attempt to upload any changes that are made to them locally.
            The service as implemented on the server is described by the CSDL document if
            all elements tagged as `Offline.ClientOnly` are removed together with all subsequently dangling references.</String>
-        </Annotation>        
+        </Annotation>
+      </Term>
+
+      <Term Name="LocalDraft" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntitySet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated model is draft and exists only on client devices. A draft entity
+        can be converted to non-draft entity for uploading"/>
+         <Annotation Term="Core.LongDescription">
+           <String>Typically for each entity type there is one corresponding entity set, however OData permits having multiple entity sets sharing the same entity type.
+            This can be taken advantage of to enable a facility for local (offline) drafts by defining an additional (draft) entity set for an existing entity type.
```

Comment:
But this differs from the existing draft concept, where drafts and active entities are in the same entity set.

Reply:
This Offline vocabulary is used to support Client Only entity types (entity types that don't exist in the backend) and Local Draft entity sets (the entity type exists in the backend but has additional entity set(s) that assist with the implementation of offline drafts). For online drafts, we wouldn't need additional entity sets.

Reply:
What can't you simply tag the entity sets that are used for local drafts as `Offline.ClientOnly`? Can there be a client-only entity set that is _not_ used for local drafts?

Reply:
There could be client-only entity set which can't be used as local drafts.
The client only entity type  and client only entity set can be defined from the client side, and the server may don't event know the existence of some client-only entity set.